public/locales/en/common.json

```diff
@@ -40,18 +40,31 @@
   "learnMoreBtn": "Learn more",
   "learnMoreAboutSCL": "Learn more about Service Canada Labs",
   "footerContactUs": "Contact us",
+  "footerContactUsURL": "/about#contact-us",
   "footerNews": "News",
+  "footerNewsURL": "https://www.canada.ca/en/news.html",
   "footerPm": "Prime Minister",
+  "footerPmURL": "https://pm.gc.ca/en",
   "footerDepartmentAgencies": "Departments and agencies",
+  "footerDepartmentAgenciesURL": "https://www.canada.ca/en/government/dept.html",
   "footerTreaties": "Treaties, laws and regulations",
-  "footerHowGovWorks": "How goverment works",
+  "footerTreatiesURL": "https://www.canada.ca/en/government/system/laws.html",
+  "footerHowGovWorks": "How government works",
+  "footerHowGovWorksURL": "https://www.canada.ca/en/government/system.html",
   "footerPublicService": "Public service and military",
-  "footerGovReporting": "Goverment-wide reporting",
-  "footerOpenGov": "Open goverment",
+  "footerPublicServiceURL": "https://www.canada.ca/en/government/publicservice.html",
+  "footerGovReporting": "Government-wide reporting",
+  "footerGovReportingURL": "https://www.canada.ca/en/transparency/reporting.html",
+  "footerOpenGov": "Open government",
+  "footerOpenGovURL": "https://open.canada.ca/en",
   "footerSocialMedia": "Social media",
+  "footerSocialMediaURL": "https://www.canada.ca",
   "footerMobileApp": "Mobile applications",
+  "footerMobileAppURL": "https://www.canada.ca",
   "footerTermsAndCondition": "Terms and conditions",
+  "footerTermsAndConditionURL": "https://www.canada.ca/en/transparency/terms.html",
   "footerPrivacy": "Privacy",
+  "footerPrivacyURL": "https://www.canada.ca/en/transparency/terms.html",
```

Comment:
Are these supposed to be the same?

Reply:
Same as above. We can get the links and add them to this PR, or add them in a separate PR, I'm indifferent to when they are updated.